RecoVertex/AdaptiveVertexFinder/plugins/InclusiveVertexFinder.h

```diff
@@ -285,11 +285,11 @@ void TemplatedInclusiveVertexFinder<InputContainer,VTX>::produce(edm::Event &eve
 #endif
                         }
 			GlobalPoint sv((*v).position().x(),(*v).position().y(),(*v).position().z());
-			float vscal = dir.unit().dot((sv-ppv).unit()) ;
-			//                        std::cout << "Vscal: " <<  vscal << std::endl;
-			if(dlen.significance() > vertexMinDLenSig  && vscal > vertexMinAngleCosine &&  v->normalisedChiSquared() < 10 && dlen2.significance() > vertexMinDLen2DSig)
+			float vscal = dir.unit().dot((sv-ppv).unit());
+			if(dlen.significance() > vertexMinDLenSig  && ( (vertexMinAngleCosine > 0) ? (vscal > vertexMinAngleCosine) : (vscal < vertexMinAngleCosine) ) &&  v->normalisedChiSquared() < 10 && dlen2.significance() > vertexMinDLen2DSig)
```

Comment:
please add some newlines to make this complex conditional more readable, esp the part related to ```(vertexMinAngleCosine > 0) ? exprA : exprB ```

Reply:
Done in https://github.com/cms-sw/cmssw/pull/23206/commits/2b642284a3f3602e2e206c6684c1397ea3748e73